src/Http/Controllers/TopicController.php

```diff
@@ -2,17 +2,20 @@
 
```

Comment:
I'd like to keep this file as it was, as well as removing the IDE helper. It's overkill to me.